src/main.rs

```diff
@@ -32,6 +33,64 @@ type PackageName<'a> = &'a str;
 type RootFeature<'a> = (PackageName<'a>, Feature<'a>);
 
 fn main() {
+    let args: Vec<String> = std::env::args().collect();
+    let args: Vec<&str> = args.iter().map(AsRef::as_ref).collect();
+
+    match args.as_slice() {
+        [_, "--stdout"] => generate_cargo_nix(io::stdout()),
```

Comment:
Use `lock` here for better performance.
```suggestion
        [_, "--stdout"] => generate_cargo_nix(io::stdout().lock()),
```

Reply:
You can also merge similar branches
```suggestion
        [_, "--stdout"] | [_, "-s"] => generate_cargo_nix(io::stdout()),
```

Reply:
That's nice. I was expecting something like that to exist.. and I tried something, and it didn't work.. and I gave up.

Thanks.